examples/bound_client.rs

```diff
@@ -0,0 +1,56 @@
+#![deny(warnings)]
```

Comment:
I think my personal preference here would be to not include an entire example file, as it means another thing to adjust when changes are required. We don't have an example for every combination of configuration options...

Reply:
No problem - it was just there to show what it would look like. I've removed it now.